cfme/tests/configure/test_register_appliance.py

```diff
@@ -88,29 +88,29 @@ def unset_org_id():
 
 
 def rhsm_unregister():
```

Comment:
These should definitely be reworked to receive the appliance object, but I think this is out of scope for the PR